leap.yaml

```diff
@@ -1,13 +1,10 @@
-codeIntegrationId: 65647e603921f60012723525
-projectId: 65647e4d3921f60012723521
+codeIntegrationId: 66c71b323f440028ad60434c
+projectId: 66c71b2e3f440028ad604348
```

Comment:
If this is changed, we cannot merge it into master (default way of use should be without experiment tracking). We can either change the name of the mapping the yaml to exp_tracking_leap.yaml and keep it or leave this in a branch